src/requests/all/edit_message_caption_inline.rs

```diff
@@ -0,0 +1,83 @@
+use serde::Serialize;
+
+use crate::{
+    network,
+    requests::{Request, ResponseResult},
+    types::{InlineKeyboardMarkup, Message, ParseMode},
+    Bot,
+};
+
+/// Use this method to edit captions of messages. On success, if edited message
+/// is sent by the bot, the edited Message is returned, otherwise True is
+/// returned.
+#[serde_with_macros::skip_serializing_none]
+#[derive(Debug, Clone, Serialize)]
+pub struct EditMessageCaptionInline<'a> {
+    #[serde(skip_serializing)]
+    bot: &'a Bot,
+
+    /// Identifier of the inline message
+    inline_message_id: String,
```

Comment:
How about to create enum `ChatOrInlineMessage` and make one method and not two? It applies to the following methods too.

Reply:
Ok.